### PR TITLE
Fix/#172

### LINKED ISSUE #172: Map Size And Background In Map View - Desktop  /  Tamaño Del Mapa Y Fondo En La Vista De Mapa - Escritorio
### Explanation:

When entering the map view, the map is small, and the background is blank.

**Current result:** 
when entering the map view, the map is small, and the background is blank.

**Expected result:** when entering the map view the map should be approximately 66% full and have rounded corners and the background should be gray #C4C4C4 with an opacity of 65%, check the vertices at larger screen sizes to make it look good.

### Screenshots:

Original: 
image

Our: image

### Checklist:

- [ ] When entering the map view the map should be occupying a space of approximately 66%
- [ ] Have rounded ends the background under the map should be gray #C4C4C4 with an opacity of 65%.
- [ ] Check the vertices at larger screen sizes to make it look good

### References:

https://www.figma.com/file/tbhMgDT10bbVjB9GazF97y/maps-proyect?node-id=407%3A46

---

# Spanish Version:

### Explicación:

Al ingresar a la vista del mapa, el mapa es pequeño y el fondo está en blanco.

**Resultado actual:**
Al ingresar a la vista del mapa, el mapa es pequeño y el fondo está en blanco.

**Resultado esperado:** al ingresar a la vista de mapa, el mapa debe estar lleno aproximadamente en un 66 % y tener esquinas redondeadas y el fondo debe ser gris #C4C4C4 con una opacidad del 65 %, verifique los vértices en pantallas de mayor tamaño para que se vea bien.

### Capturas de pantalla:

Original:
imagen

Nuestras: imagen

### Lista de Verificación:

- [ ] Al ingresar a la vista de mapa, el mapa debe estar ocupando un espacio de aproximadamente 66% .
- [ ] Tener extremos redondeados, el fondo debajo del mapa debe ser gris #C4C4C4 con una opacidad del 65%.
- [ ] Comprobar los vértices en pantallas de mayor tamaño para que se vea bien.

### Referencias:

https://www.figma.com/file/tbhMgDT10bbVjB9GazF97y/maps-proyect?node-id=407%3A46

### PR DESCRIPTION
**Pull request by mdrianislam0or1**
### Description
 Map Size And Background In Map View - Desktop / Tamaño Del Mapa Y Fondo En La Vista De Mapa - Escritorio #172 

#### Screenshots
fix#172

### CODE REVIEW

**Comment by angelik0828** [comment]
@mdrianislam0or1 
1) The text used in the localization is not bold and is not proportionally sized.
2)the text used in "estimated time of arrival" is not bold and should not be as large

**Review by Israel-Laguan — APPROVED** [review]

File: src/pages/Delivery/Delivery.layout.jsx [hunk 1]
```diff
@@ -1,23 +1,15 @@
...
+import { Fragment } from 'react'
 import assistant from '../../assets/assistant.png'
 import chat from '../../assets/chat.png'
 import gps from '../../assets/gps-icon.png'
-import { Fragment } from 'react'
+import { Text } from '../../components'
+import {
+  ChatBtn, ChatBtnContainer, FooterMap,
+  FooterMapAddress, HeaderMap, HeaderSubTitle, HeaderText, HeaderTitle, MapContainer, MapLayoutContainer
+} from './Delivery.styled'
```
Comment (heliomar-pena) [inline src/pages/Delivery/Delivery.layout.jsx]: Please check if you can run Prettier or similar to sort correctly the imports, I think is more readable one on top of the other

File: src/pages/Delivery/Delivery.layout.jsx [hunk 2]
```diff
@@ -66,12 +58,12 @@ const Delivery = ({
           >
             <img src={gps} width="22px" height="31px" alt="Gps icon" style={{ margin: '10px'}} />
           </FooterMapAddress>
-          <FooterMapAddress cols={'0 0 50%'} txt={'Tiempo estimado de llegada'}>
-            <Text medium bold margin="8" style={{ textTransform: 'lowercase', color:'#003B62',fontWeight:'700'}}>
+          <FooterMapAddress style={{fontSize:'25px'}} cols={'0 0 50%'} txt={'Tiempo estimado de llegada'}>
+            <Text medium bold margin="8" style={{ textTransform: 'lowercase',fontSize:'23px', color:'#003B62',fontWeight:'700'}}>
```
Comment (heliomar-pena) [inline src/pages/Delivery/Delivery.layout.jsx]: please leave the proper space between each attribute, mm I noticed that sometimes it has space and sometimes not, please check what is the best standard for this case, or we could make a poll hehe.

I like more with the spaces

**Review by heliomar-pena — APPROVED** [review]
please check my last comment and it's ready to go

**Review by e-azocar — APPROVED** [review]

**Comment by angelik0828** [comment]
@mdrianislam0or1  
The map view is checked I think they are missing:

1. Some margins on the sides.
2. The gray color in the background as mentioned in the issue.
please check the list items for the issue

image


image

**Review by angelik0828 — APPROVED** [review]

**Comment by arodriguezl9302** [comment]
@mdrianislam0or1 please resolve conflicts

**Pull request merged**